build-bootstrap.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const autoprefixer = require('autoprefixer');
+const postcssPresetEnv = require('postcss-preset-env');
+const { writeFile } = require('fs');
+const { render: renderScss } = require('node-sass');
+const postcss = require('postcss');
+
+const inFile = 'keepassxc-browser/styles/bootstrap.scss';
+const outFile = 'keepassxc-browser/styles/bootstrap.css';
+
+renderScss(
```

Comment:
If we are going to use the build script, the CSS output file should be minimized. Also, I'm not sure if the extension is going to pass the AMO/CWB audit if there's .scss files inside the package.

Reply:
The AMO Wiki [doesn't recommend minifying code](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/Source_Code_Submission):

> Obfuscating or minifying your code isn't recommended. It rarely protects your code, as these techniques can be reversed using tools such as JSNice and JS Beautifier. And, unlike the advantage that minified code offers web pages loaded over the internet, as extension code is loaded from a local source the performance benefits are not significant.

[According to the AMO Wiki](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/Source_Code_Submission#Provide_your_extension_source_code) the `bootstrap.scss` file shouldn't present a problem:

> ## When must source code be uploaded?
> You must upload your extension’s source code when its code was created using:
> [...]
> * any other custom tool that takes files, applies pre-processing, and generates file(s) to include in the extension.

However it would be necessary to provide build instructions and [upload the sources when publishing a new addon version](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/Source_Code_Submission#How_to_upload_source_code):

> ## Provide build instructions
> To reproduce the build, the reviewer runs the instructions you provided and then uses a diff tool to compare the generated sources to those in the extension. There must be no differences. The easiest way to provide the build instructions is to include a README file with the submitted source code. [...]
>
> [...]
> When using npm, yarn, or other package management tools that support it, be sure to include the lockfile, for example, `package-lock.json`. Otherwise, reviewers may use a different version resulting in differences between the generated code and that in the extension.
> 
> [...] However, you don’t need to describe how to install common tools such as npm or node.

Reply:
Still, the .scss file is not needed inside the extension.